packages/plugin-infra/src/types.ts

```diff
@@ -644,10 +644,17 @@ export namespace Plugin.SNSAdaptor {
         FocusingPost = 'focusing-post',
         Post = 'post',
         Profile = 'profile',
+        MirrorMenu = 'mirror-menu',
+        MirrorEntry = 'mirror-entry',
+    }
+    export interface TipsAccount {
+        address: string
+        name?: string
     }
     export interface TipsRealmOptions {
         identity?: ProfileIdentifier
         slot: TipsSlot
+        tipsAccounts?: TipsAccount[]
```

Comment:
I will change it to `recipients` in https://github.com/DimensionDev/Maskbook/pull/7474.